### PR TITLE
fix: fix bitcoin fee rate and increase evm outtx inclusion timeout

### PR DESCRIPTION
**Pull request by ws4charlie**
# Description
The economical fee rate `sometimes` can be half of the conservative fee rate (forgot about taking a screenshot for that. There other day was around `49sats/vB` vs. `26 sats/vb`).

image

monitored a bit more
image
image

blocks like 829506, 829530, the `Fee Span` range was quite wide and `Conservative` gas rate was `3~4` times of
`Economical` rate.
image
image

Closes: [1707](https://github.com/zeta-chain/node/issues/1707)
Closes: [1712](https://github.com/zeta-chain/node/issues/1712)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [x] Tested CCTX in localnet
- [ ] Tested in development environment
- [x] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: zetaclient/bitcoin_client_test.go [hunk 1]
```diff
@@ -158,6 +175,63 @@ func (suite *BitcoinClientTestSuite) Test3() {
...
+func TestBitcoinFeeRate(t *testing.T) {
+	// mainnet config
+	connCfg := &rpcclient.ConnConfig{
+		Host:         "127.0.0.1:8332",
+		User:         "username",
+		Pass:         "password",
+		Params:       "mainnet",
+		HTTPPostMode: true,
+		DisableTLS:   true,
+	}
```
Comment (lumtis) [inline zetaclient/bitcoin_client_test.go:192]: We can't use it for unit test
What about creating a mock for the client?
It doesn't seem we need anything more than a mock for `EstimateSmartFee`

Comment (ws4charlie) [reply zetaclient/bitcoin_client_test.go:192]: [commit](https://github.com/zeta-chain/node/pull/1713/commits/7ebaf3805a1dd7284917117e7e9b2f009c2fe037)

Comment (lumtis) [reply zetaclient/bitcoin_client_test.go:192]: Can we add a comment how this test is supposed to be run? We might consider adding this to the E2E test lists?

**Review by lumtis — APPROVED** [review]
LGTM, just one comment on the test

File: zetaclient/bitcoin_client_test.go (hunk 1 above)
Comment (ws4charlie) [reply zetaclient/bitcoin_client_test.go:192]: I tried it on the Bitcoin testnet. The API returned me same number regardless of what `confTarget` `estimateMode` passed. What we expect in the test case is that the higher the `confTarget` passed, the lower the rate; and `Conservative` must give a higher gas rate than `Economical` if same `confTarget` is given. These two assertions will break in Testnet (I guess private network will also break the assertions).
image

**Comment by ws4charlie** [comment]
> LGTM, just one comment on the test

The way I used to run `LiveTestSomeTest` is to remove the prefix `Live` of the function (`TestBitcoinFeeRate` in this case) to make it an executable unit test. It's not an ideal way TBH.

**Review by brewmaster012 — APPROVED** [review]

**Pull request merged**